cheroot/test/test_ssl.py

```diff
@@ -149,10 +150,15 @@ def make_tls_http_server(bind_addr, ssl_adapter, request):
     return httpserver
 
 
+def get_key_password():
+    """Provide hardcoded password for private key as callable."""
```

Comment:
This is a bit misleading because this function does not actually return a callable. It returns a string.
```suggestion
    """Return a predefined password string.

    It is to be used for decrypting private keys.
    """
```

Reply:
Done in latest push